client/src/views/Simulation/components/ModelPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <section class="simulation-model-panel">
+    <settings-bar>
+      <counters slot="left" :title="modelName" :data="countersData" />
+      <aside slot="right">
+        <div class="dropdown">
+          <button
+            class="btn btn-primary dropdown-toggle"
+            type="button"
+            @click="settingsOpen = !settingsOpen">
+            Settings
+          </button>
+          <div class="dropdown-menu dropdown-menu-right" :class="{ 'show': settingsOpen }">
```

Comment:
Maybe we want to make these settings panel closer to how we do it in the `Model` view:
![image](https://user-images.githubusercontent.com/10552785/122445774-dcf07e00-cf6f-11eb-83f4-dc41ae2cf450.png)

Reply:
I did not do that because in the future GroMet will support many different format, and that design will not be quickly updated when adding those formats.